docs/domains/Domain.json

```diff
@@ -562,12 +564,66 @@
           "description": "Name of a ConfigMap containing the WebLogic Deploy Tooling model.",
           "type": "string"
         },
+        "onlineUpdate": {
+          "description": "Online update option for Model In Image dynamic update.",
+          "$ref": "#/definitions/OnlineUpdate"
+        },
         "modelHome": {
           "description": "Location of the WebLogic Deploy Tooling model home. Defaults to /u01/wdt/models.",
           "type": "string"
         }
       }
     },
+    "OnlineUpdate": {
+      "type": "object",
+      "properties": {
+        "deployTimeoutMilliSeconds": {
+          "description": "WLST deploy application or libraries timout in milliseconds. Default: 180000.",
```

Comment:
(1) The term "WLST" in each  of these descriptions is confusing, as that's technically an internal detail of the MII/WDT implementation.
(2) I personally prefer "Millis" to "MilliSeconds".  It's shorter and more common (at least it's more common in WL mbean attribute names).
(3) Maybe we can have a meeting to word-smith each of these settings?

Reply:
(4) Because OnlineUpdate has so many timeouts, maybe they can become a subgroup of OnlineUpdate? This would make a 'kubectl explain' prettier.  As in "OnlineUpdate.Timeouts.deployTimeoutMillis", etc.

Reply:
These are good ideas... I'm not opposed to wordsmithing prior to releasing a schema that we will be stuck with :).

Reply:
moved all timeouts under wdtTimeouts